make.py

```diff
@@ -160,7 +160,7 @@ def run(self, cwd: Path, extra_cmd_args: Iterable[str]) -> None:
         # (note only the first `maturin develop` is impacted, as all maturin
         # develop uses the same default virtualenv)
         Cmd(
-            cmd="poetry install --with=docs,testbed-server",
+            cmd="poetry install --with=testbed-server",
```

Comment:
No need to install docs in `make.py`? btw, maybe we should create an issue to add a `$ ./make.py docs` option.

Reply:
Both `pyproject.toml` with poetry will not share the same `virtualenv`.

IMO very few persons need the docs dependencies so maybe a `make.py docs` that do a `poetry -C docs install` could do the trick

Reply:
I've created the issue #5356 to track that